testutil/singlevm/setup.sh

```diff
@@ -207,6 +207,27 @@ sudo cp -f clear-"${LATEST}"-cloud.img /var/lib/ciao/images
 cd /var/lib/ciao/images
 sudo ln -sf clear-"${LATEST}"-cloud.img df3768da-31f5-4ba6-82f0-127a1a705169
 
+# Set macvlan interface
+if [ -x "$(command -v ip)" ]; then
```

Comment:
you may want to check if the network is already setup and skip this step.
And user may run this script multiple times in the course of his/her workflow.